students/Kollii/lesson05/except_exercise.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/python
+
+"""
+An exercise in playing with Exceptions.
+Make lots of try/except blocks for fun and profit.
+
+Make sure to catch specifically the error you find, rather than all errors.
+"""
+
+from except_test import fun, more_fun, last_fun
+
+
+# Figure out what the exception is, catch it and while still
+# in that catch block, try again with the second item in the list
+first_try = ['spam', 'cheese', 'mr death']
+
+try:
+    joke = fun(first_try[0])
+except NameError:
+    print('Name is not defined')
```

Comment:
> while still in that catch block, try again with the second item in the list

you caught the exception but the instructions says to try again but with the second item in the list